custom/icds_reports/templates/icds_reports/icds_app/navigation.directive.html

```diff
@@ -131,6 +131,12 @@
                 </ul>
             </li>
             <hr>
+            <li ng-class="{'active': $location.path().indexOf('/service_delivery_dashboard') !== -1}" ng-if="haveAccessToAllLocations === true">
```

Comment:
I think it should be `haveAccessToFeatures` instead of `haveAccessToAllLocations`?

Reply:
Yes, you are right. Great catch.